tests/spec/core/xref-spec.js

```diff
@@ -198,9 +203,9 @@ describe("Core — xref", () => {
 
   it("uses data-cite to disambiguate", async () => {
     const body = `
-      <section id="links" data-cite="service-workers">
-        <p><a>fetch</a> is defined 1 time in service-workers and 2 times in fetch.
-          It uses parent's data-cite (service-workers).</p>
+      <section id="links" data-cite="css-snappoints">
```

Comment:
Why this and some others switched to another spec?

Reply:
Due to https://github.com/sidvishnoi/respec-xref-route/pull/41

By default, respec doesn't send a for context (unless specified), causing to return unnecessary ambiguity errors - [Example](https://user-images.githubusercontent.com/8426945/61529581-5196a700-aa3f-11e9-80a2-f54eebae9bcc.png)

`fetch` is implicitly (without a `forContext`) defined only in [Fetch Standard](https://fetch.spec.whatwg.org/#concept-fetch). But earlier, xref API returned [3 results for it](https://user-images.githubusercontent.com/8426945/61529500-1a27fa80-aa3f-11e9-87ef-e659da0b25f6.png). Same reason in other tests.

Reply:
👍 👍 👍